src/Login.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Login Successful
+  Scenario: User should be able to login successfully with valid username and password.
+    Given User is on Home Page of given website
+    When user clicks on Sign In Page
+    And enters Valid Credentials
+    Then he should be able to see the welcome page
```

Comment:
I think you need to cover more scenarios min 7 e.g 
Login with invalid username and valid password 
Login with valid username and invalid password 
Login with empty username and valid password etc.

Reply:
I have added 7 more scenarios to the original one.